### PR TITLE
feat: added Slider

### PR DESCRIPTION
**Pull request by AnjaliDeshwani**
**Slider**
- slider.html
- slider.css

Preview of both available at- https://deploy-preview-11--winsome-ui.netlify.app/components/slider/slider

### CODE REVIEW

File: components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+.slider:hover {
+  opacity: 1;
+}
+
+.slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 25px;
+  height: 25px;
```
Comment (Nishant-96) [inline components/Slider/slider.css]: Try to use relative units.

**Review by Nishant-96 — APPROVED** [review]

File: components/Slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.slider-container {
+  width: 100%;
+}
+
+.slider {
+  -webkit-appearance: none;
+  width: 100%;
+  height: 15px;
+  background: #d3d3d3;
```
Comment (ikkurthis1998) [inline components/Slider/slider.css]: I think you missed using color variable here

**Review by ikkurthis1998 — APPROVED** [review]

File: components/Slider/slider.css (hunk 1 above)
Comment (AnjaliDeshwani) [reply components/Slider/slider.css]: Thanks Nishant for pointing it out, I have updated now.

File: components/Slider/slider.css (hunk 2 above)
Comment (AnjaliDeshwani) [reply components/Slider/slider.css]: Yes right, thank you. I have updated now.

**Review by Tejs1 — COMMENTED** [review]
lgtm

**Pull request merged**